### ISSUE TITLE
expose paramString to macro

### ISSUE THREAD

**Issue by tobibeer** [Jermolene/TiddlyWiki5#1373]
pass the entire **paramString** as a variable containing all params as a string to the macro context, e.g.

```
\define my-macro()
<<paramString>>

; outputs
: `foo, bar, baz`
\end

<<my-macro foo, bar, baz>>
```

**Comment by tobibeer** [comment]
For lack of documentation, I couldn't figure out if this is possibly covered by https://github.com/Jermolene/TiddlyWiki5/pull/277

**Comment by Skeeve** [comment]
No. It isn't. It's just for JavaScript Macros. For wikitext macros more tweaking would be required, But I think I explicitly stated it in my #270 "It's merely for the javascript based macros. Simply if we have defined the function without arguments"

**Comment by tobibeer** [comment]
Ah, right. Makes sense. Thanks for pointing out.

**Comment by tobibeer** [comment]
Closed, since #1398 was rejected.
